generate.sh

```diff
@@ -13,20 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: Write a Python script to build the docs.
-# * List all tarballs and find docfx-* ones.
+# TODO:
 # * If FORCE_GENERATE_ALL env var is set to true, regenerate all.
-# * If SOURCE_FILE_GS_URI is set, force regeneration for that file.
-# * For each relevant tarball:
-#   1. Create a temp directory, and an obj directory inside.
-#   2. Unpack the tarball inside the obj directory.
-#   3. Create a docfx.json file (using title=package name and
-#      _rootPath=serving path).
-#   4. docfx build -t ...
-#   5. Move to the output directory, copy in obj/docs.metadata, and call
-#      docuploader upload.
+# * If SOURCE_FILE_NAME is set, force regeneration for that file.
 
 set -e
 
-# Now it's only making sure `docfx --help` works
-docfx --help
+if [ -z "$SOURCE_BUCKET" ]; then
+    echo "Must set SOURCE_BUCKET"
+    exit 1
+fi
+
+# Add the path where docuploader gets installed to PATH.
+export PATH=$PATH:/h/.local/bin
```

Comment:
`${HOME}/.local/bin` is bit better because it will continue to work even if we change the home directory path.

Reply:
Done.